builtin/logical/postgresql/secret_creds.go

```diff
@@ -138,6 +151,10 @@ func (b *backend) secretCredsRevoke(
 		pq.QuoteIdentifier(username)))
 
 	revocationStmts = append(revocationStmts, fmt.Sprintf(
```

Comment:
@sean- pg.QuoteIdentifier should defend against 'ole Bobby, but i will take a look at it.